Modules/_elementtree.c

```diff
@@ -283,10 +283,12 @@ create_new_element(PyObject* tag, PyObject* attrib)
     PyObject_GC_Track(self);
 
     if (attrib != Py_None && !is_empty_dict(attrib)) {
+        attrib = PyDict_Copy(attrib);
```

Comment:
Why this change has been added?

Reply:
Because the results of the tests in this commit uncovered a discrepancy between the C and Python implementations whereby the attrib list is not copied in the C version beyond the `__init__` function. (The Python implementation always uses `__init__` whenever it creates a new element, but the C implementation uses `create_new_element` in all locations except the original `__init__` function.)

Reply:
In many case this makes a redundant copying since `create_new_element` is called after copying a dict.

Reply:
Can this be moved out of this function into the caller(s) that need it?